### PR TITLE
Update project package versions

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

## Effect

The following changes are already committed:

- resolves `ERROR #98123  WEBPACK`:

```
 ERROR #98123  WEBPACK

Generating SSR bundle failed

[BABEL] ~/projects/documentation/.cache/develop-static-entry.js: Cannot find module '@babel/compat-data/corejs3-shipped-proposals'
Require stack:
```

ran this in the repo's root to resolve:

```
npm install -g --save core-js@^3
```

- update packages with npm

```
npm update
```

## Remaining Work

The following changes still need to be completed:

- [x] Technical edit

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: src/components/resourceSelector.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { useStaticQuery } from "gatsby"
 import Partial from "./partial"
 import propTypes from "prop-types"
 import withLocation from "./withLocation"
+import { graphql } from "gatsby"
```
Comment (EdwardAngert) [inline src/components/resourceSelector.js:6]: this was suggested (not required) by a warning I was getting:

```
warn Using the global `graphql` tag is deprecated, and will not be supported in v3.
Import it instead like:  import { graphql } from 'gatsby' in file:
/Users/workward/projects/documentation/src/components/resourceSelector.js
```

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5655 | Difference |
| --- | --- | --- | --- |
| Performance Score | `46` | `36` | `-10` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5655` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20186/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20186/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: src/components/resourceSelector.js (hunk 1 above)
Comment (alexfornuto) [reply src/components/resourceSelector.js:6]: This warning is coming from other files as well, but it doesn't always specify where like it does here.

**Comment by alexfornuto** [comment]
Added `Needs More Info` because the PR template is empty.

**Pull request merged**